tests/test_configurable_lookback_window.py

```diff
@@ -0,0 +1,86 @@
+"""Test tap configurable properties. Specifically the lookback_window"""
+import re
+import os
+from datetime import datetime as dt
+from datetime import timedelta
+
+from tap_tester import menagerie, connections, runner
+
+from base import BaseTapTest
+
+
+class ConversionWindowBaseTest(BaseTapTest):
+    """
+    Test tap's sync mode can execute with valid lookback_window values set.
+    Validate setting the lookback_window configurable property.
+    Test Cases:
+    Verify connection can be created, and tap can discover and sync with a lookback window
+    when passed in config else takes default value.
+    """
+    lookback_window = ''
+
+    def name(self):
+        return f"tt_stripe_lookback_window_{self.lookback_window}"
+
+    def get_properties(self):
+        """Configurable properties, with a switch to override the 'start_date' property"""
+        return_value = {
+            'start_date':dt.strftime(dt.utcnow() - timedelta(seconds=600), self.START_DATE_FORMAT),
+            'lookback_window': self.lookback_window,
+            'account_id': os.getenv('TAP_STRIPE_ACCOUNT_ID'),
+            'client_scret': os.getenv('TAP_STRIPE_CLIENT_SECRET')
```

Comment:
@namrata270998 Can you please correct the spell of client_secret here because the test is failing for this reason? Also, this test doesn't have any assert statement so what are we checking here?

Reply:
This test was suggested by Kyle on the other PR for balance transactions lookback [here](https://github.com/singer-io/tap-stripe/pull/125#pullrequestreview-900330370). It is just to check that the lookback window is added as a UI parameter or not